### ISSUE TITLE
Refactor the PheWAS module

### ISSUE THREAD

**Issue by MichaelaEBI** [opentargets/platform#195]
Tidying up the PheWAS code, addressing the following points:
- [ ] Ideally read and write files to google bucket(s). Files that need to be read are:
 - The data file from phewas.org: phewas-catalog.csv
 - Mapping file from phewas.org: phecode_icd9_map_unrolled.csv.zip
- OT mapping file from the mapping GitHub repository: [phewascat.mappings.tsv](https://github.com/opentargets/mappings/blob/master/phewascat.mappings.tsv)
- [x] Filter the evidence strings based on p-value (< 0.05) while reading in the phewas-catalog.csv file
- [x] Move phewas-related settings back into settings.py file (from constants.py)
- [ ] Consolidate run.py and make_efo_mapping.py into a single phewas.py file

**Comment by ElaineMcA** [comment]
Moved to 19.04 - post EFO 3 work.

**Comment by afaulconbridge** [comment]
Moving under "Roadmap: Harmonize evidence_datasource_parsers" #535

**Comment by AsierGonzalez** [comment]
The PheWAS catalog parser has been revamped (see PR [#44](https://github.com/opentargets/evidence_datasource_parsers/pull/44)) and the main changes are the following:

- Parser is now called `PheWAS.py` and is located directly in the `modules` directory. The `phewascat` directory in `modules` an all the files and scripts underneath it have been deleted.
- Input and output file names are specified in `settings.py` instead of being hardcoded in module.
- The input csv file has to be local
- The Phecode to EFO mapping file is read from the GitHub mappings repo.
- Odds ratio and number of cases have been moved to the evidence section from the unique_association_fields.
- Target and disease ids have been added to unique_association_fields

In a nutshell, the parser has been simplified given that now that the Phecodes to EFO mappings have been completed (see [#420](https://github.com/opentargets/platform/issues/420)) there is no need to use complementary files for the mapping.

The only thing that is left to be done is to refine the unique association fields to reduce the number of duplicate evidence strings. That work can be followed in [#795](https://github.com/opentargets/platform/issues/795).
